typescript/src/e2e.test.ts

```diff
@@ -208,6 +208,27 @@ describe("CDP Client E2E Tests", () => {
     expect(account.name).toBe(randomName);
   });
 
+  it("should create an end user with EVM smart account and Solana account", async () => {
+    const randomEmail = `test-${Date.now()}@example.com`;
+
+    const endUser = await cdp.endUser.createEndUser({
+      authenticationMethods: [{ type: "email", email: randomEmail }],
+      evmAccount: { createSmartAccount: true },
+      solanaAccount: { createSmartAccount: false },
```

Comment:
why is `createSmartAccount` an option on solanaAccount?

Reply:
Yeah we discussed that trade off before.
The empty body was more a bit more awkward to avoid than the smart wallet confusion. At least here, the EVM and Solana account objects a bit more consistent.